object-cache.php

```diff
@@ -508,64 +554,171 @@ function switch_to_blog( $blog_id ) {
 		$this->blog_prefix = ( is_multisite() ? $blog_id : $table_prefix );
 	}
 
-	function colorize_debug_line( $line ) {
+	function colorize_debug_line( $line, $trailing_html = '' ) {
 		$colors = array(
 			'get' => 'green',
-			'get_multi' => 'green',
+			'get_local' => 'lightgreen',
+			'get_multi' => 'fuchsia',
 			'set' => 'purple',
+			'set_local' => 'orchid',
 			'add' => 'blue',
 			'delete' => 'red',
+			'delete_local' => 'tomato',
+			'slow-ops' => 'crimson',
 		);
 
 		$cmd = substr( $line, 0, strpos( $line, ' ' ) );
 
 		// Start off with a neutral default color...
 		$color_for_cmd = 'brown';
-
 		// And if the cmd has a specific color, use that instead
 		if ( isset( $colors[ $cmd ] ) ) {
 			$color_for_cmd = $colors[ $cmd ];
 		}
 
-		$cmd2 = '<span style="color:' . esc_attr( $color_for_cmd ) . '">' . esc_html( $cmd ) . '</span>';
+		$cmd2 = "<span style='color:{$color_for_cmd}; font-weight: bold;'>" . esc_html( $cmd ) . "</span>";
```

Comment:
Restore the `esc_attr` just to be safe:

```suggestion
		$cmd2 = "<span style='color:" . esc_attr( $color_for_cmd ) . "; font-weight: bold;'>" . esc_html( $cmd ) . "</span>";
```

Reply:
I tend not to do this for values we set ourselves, but yeah, better safe...